src/Stratis.Bitcoin.Features.RPC.Tests/Controller/FullNodeControllerTest.cs

```diff
@@ -687,5 +687,52 @@ private byte[] GetBytes(uint compact)
                 (byte)(compact)
             };
         }
+
+        public class TestReadOnlyNetworkPeerCollection : IReadOnlyNetworkPeerCollection
```

Comment:
Where is this used?

Reply:
Somewhere up in that test class I think, copied it here because it was used from the reference that was taken away but it can probably just as easily be a mock instead.